src/renderer/components/commands-action-button.tsx

```diff
@@ -409,9 +408,11 @@ export class GistActionButton extends React.Component<
   };
 
   private gistFilesList = (values: EditorValues) => {
-    const ids = [...DEFAULT_EDITORS, ...Object.keys(values)];
+    values = ensureRequiredFiles(values);
```

Comment:
nit(non-blocking): any reason for re-assigning `values` here rather than creating a new `const` variable?